docs/content/getting-started/python.md

```diff
@@ -72,23 +73,29 @@ positions[:,0] = np.linspace(-10,10,10)
 colors = np.zeros((10,3), dtype=np.uint8)
 colors[:,0] = np.linspace(0,255,10)
 
-rr.log_points("my_points", positions=positions, colors=colors, radii=0.5)
+rr.log(
+    "my_points",
+    rr.Points3D(positions, colors=colors, radii=0.5)
+)
 ```
 
 When you run this script you will again be greeted with the [Rerun Viewer](../reference/viewer/overview.md), this time
 only showing a simple line of red points.
 
 <picture>
-  <source media="(max-width: 480px)" srcset="https://static.rerun.io/quickstart1_line/37d42194bc99cbb805e3ca53eba11c2896616893/480w.png">
-  <source media="(max-width: 768px)" srcset="https://static.rerun.io/quickstart1_line/37d42194bc99cbb805e3ca53eba11c2896616893/768w.png">
-  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/quickstart1_line/37d42194bc99cbb805e3ca53eba11c2896616893/1024w.png">
-  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/quickstart1_line/37d42194bc99cbb805e3ca53eba11c2896616893/1200w.png">
-  <img src="https://static.rerun.io/quickstart1_line/37d42194bc99cbb805e3ca53eba11c2896616893/full.png" alt="simple line">
+  <img src="https://static.rerun.io/quickstart1_line/969a72007a8354bc2638d5794fa7d47176135d0c/full.png" alt="">
+  <source media="(max-width: 480px)" srcset="https://static.rerun.io/quickstart1_line/969a72007a8354bc2638d5794fa7d47176135d0c/480w.png">
+  <source media="(max-width: 768px)" srcset="https://static.rerun.io/quickstart1_line/969a72007a8354bc2638d5794fa7d47176135d0c/768w.png">
+  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/quickstart1_line/969a72007a8354bc2638d5794fa7d47176135d0c/1024w.png">
+  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/quickstart1_line/969a72007a8354bc2638d5794fa7d47176135d0c/1200w.png">
 </picture>
 
 
-The [rr.log_points](https://ref.rerun.io/docs/python/latest/common/spatial_primitives/#rerun.log_points) function can
-take any Nx2 or Nx3 numpy array as a collection of positions.
+
+The [`rr.log`](https://ref.rerun.io/docs/python/HEAD/common/logging/#rerun.log) is the primary way to log data. It's a flexible
+call that can accept a variety of correctly-formatted data—including your own. The easiest way to use it is to use an instance of
+one of the built-in *archetype* class, such as [`rr.Points3D`](https://ref.rerun.io/docs/python/HEAD/common/spatial_primitives/#rerun.Points3D). Archetypes take care of gathering the various
+components representing, in this case, a batch of 3D points such that it is recognized and correctly displayed by the Rerun viewer. The `rr.Points3D` archetype accepts any collection of positions that can be converted to a Nx3 Numpy array, along with other components such as colors, radii, etc.
```

Comment:
good blueprint for the missing method documentation :)

Reply:
Yes. BTW: https://github.com/rerun-io/rerun/issues/3624